crates/type/src/digest/mod.rs

```diff
@@ -40,3 +42,10 @@ impl From<base64::DecodeError> for Error {
         Self::Decode(value)
     }
 }
+
+pub async fn hash_data(data: &[u8]) -> ContentDigest {
```

Comment:
https://github.com/profianinc/drawbridge/pull/132

Reply:
please use the added method

Reply:
Sorry, forgot to delete it :)